# è‡ªåŠ¨å¯¼å…¥å›½é™…åŒ–åŒ…è£¹å‡½æ•° (`i18n-wrap/import-i18n-function`)

ğŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```

### Options

If there are any options, describe them here. Otherwise, delete this section.

```js

  auto: {
    type: 'boolean',
    default: false, // é»˜è®¤å€¼
    description: 'æ˜¯å¦è‡ªåŠ¨å¯¼å…¥å›½é™…åŒ–åŒ…è£¹å‡½æ•°',
  },
  pathSuffix: {
    type: 'string',
    default: DEFAULT_PATH_SUFFIX, // é»˜è®¤å€¼
    description: 'å›½é™…åŒ–åŒ…è£¹å‡½æ•°çš„è·¯å¾„åç¼€',
  },
  srcAlias: {
    type: 'string',
    default: DEFAULT_SRC_ALIAS, // é»˜è®¤å€¼
    description: 'æºä»£ç çš„åˆ«å',
  },
  staticEntry: {
    type: 'string',
    default: DEFAULT_STATIC_ENTRY, // é»˜è®¤å€¼
    description: 'é™æ€èµ„æºçš„å…¥å£æ–‡ä»¶',
  },

```

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
